webpack.config.js

```diff
@@ -53,6 +53,11 @@ Encore
         config.corejs = 3;
     })
 
+    .copyFiles({
```

Comment:
I'm not sure how I'm going to use Vue to load the `loading.gif` image in the template. Since Vue templates don't have the twig `asset` functionality. The solution to me looks quite complicated any way I look at it. Should we discuss it more? Or just leave it in without the hash?

Reply:
I was hoping you'd ask that! In Vue (or really, in any JavaScript file), you can *import* the image into variable then use it. Something like this (written in pure JS for simplicity):

```
const loadingGif = import '../images/loading.gif';

console.log(`<img src="${loadingGif}">`);
```

Or if you're in CSS, just use the relative path to the image:

```
.loading {
    background-image: url('../images/loading.png');
}
```

This is a Webpack feature (well, a `file-loader` feature in Webpack) and both of these are identical in Webpack's eyes. In both cases, the final path to the image - including any potential version hashes - will be used. There's a bit more info about this approach here: https://webpack.js.org/guides/asset-management/#loading-images